### PR TITLE
🎈Added Kijk (1.0.0)

### LINKED ISSUE #3047: Kijk | kijk.nl
**Prerequisites and essential questions**
- [x] Is it a popular site?
- [x] Is the website older than 2 months?
- [x] Is the site locked to a specific country/region? [I don't know if it's geo-blocked for sure, but I assume it is. It's a Dutch website)
- [ ] Is the site a paid service? (e.g. Netflix, Hulu) (yes and no, there's free content and after certain amount of days they are only viewable with a subscription. Some older stuff is just free to watch. Much like NPO Start (free version), NPO Plus (paid version/RTL XL (paid and free) in the Netherlands, and it would be cool if those could be added too! Same answers apply.)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** lifeiscrazy『marr』#3604

**What's the name of the service?** https://kijk.nl

**What should the Presence display?** Like with Netflix. The movie or series name, episode number, remaining time

**If possible, please provide a logo for the service (512x512 minimum)**

SVG logo link: https://images.ctfassets.net/0w1d9hjfek5f/3FZroCvgcgYMGmaImuIKEK/356ce7a5b35fc83431a75155bb358693/brand-kijk2.0.svg

### PR DESCRIPTION
**Pull request by DarkVillager**
Resolves #3047 
Added kijk basic functionality:
- Added Browsing
- Added Homepage
- Added Watching Tv Shows
- Added Watching Movies
- Added Watching Missed Tv Shows

Screenshot_4
Screenshot_5
Screenshot_6
Screenshot_7
Screenshot_8
Screenshot_9
Screenshot_10

### CODE REVIEW

File: websites/K/Kijk/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Dark_Ville",
+    "id": "638080361179512853"
+  },
+  "url": "www.kijk.nl",
+  "description": {
+    "nl": "Kijk opent je ogen",
+    "en": "Kijk opens your eyes."
```
Comment (ririxidev) [inline websites/K/Kijk/dist/metadata.json:10]: is it really their desc?

File: websites/K/Kijk/presence.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      if (!search) {
+        presenceData.startTimestamp = browsingStamp;
+        presenceData.details = "Viewing the homepage";
+      } else if (search.value !== "") {
+        presenceData.startTimestamp = browsingStamp;
+        presenceData.details = "Searching For:";
+        presenceData.state = search.value;
+        presenceData.smallImageKey = "searching";
+      } else { 
+        presenceData.startTimestamp = browsingStamp;
```
Comment (ririxidev) [inline websites/K/Kijk/presence.ts:28]: why you set ``startTimestamp`` every time when you already set it before?

File: websites/K/Kijk/presence.ts [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    }
+    if (page === "/programmas" || page === "/programmas/") {
+      presenceData.details = "Discovering:";
+      presenceData.state = "Programma's";
+      presenceData.startTimestamp = browsingStamp;
+    }
+    if (page === "/films" || page === "/films/") {
+      presenceData.details = "Discovering:";
+      presenceData.state = "Films";
+      presenceData.startTimestamp = browsingStamp;
```
Comment (ririxidev) [inline websites/K/Kijk/presence.ts:40]: above ^

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/K/Kijk/dist/metadata.json (hunk 1 above)
Comment (DarkVillager) [reply websites/K/Kijk/dist/metadata.json:10]: Yep sadly it is.

File: websites/K/Kijk/presence.ts [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo"
+  },
+  page = window.location.pathname,
+  host = document.location.hostname; 
+
+    presenceData.startTimestamp = browsingStamp;
+    if (host === "www.kijk.nl") {
```
Comment (Bas950) [inline websites/K/Kijk/presence.ts:16]: this if is useless?

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/K/Kijk/presence.ts (hunk 4 above)
Comment (DarkVillager) [reply websites/K/Kijk/presence.ts:16]: For now yes.

**Review by Bas950 — CHANGES_REQUESTED** [review]
You should probs just use Dutch details, instead of Viewing like Bekijkt or something like that, since I am pretty sure this website is basically dutch only :/

**Comment by DarkVillager** [comment]
> You should probs just use Dutch details, instead of Viewing like Bekijkt or something like that, since I am pretty sure this website is basically dutch only :/

Sure

**Review by Bas950 — APPROVED** [review]

**Review by StrikerFRFX — APPROVED** [review]

**Pull request merged**